sync-core/src/test/java/com/cloudant/sync/replication/ReplicationTestBase.java

```diff
@@ -37,6 +37,8 @@
     protected CouchClientWrapper remoteDb = null;
     protected CouchClient couchClient = null;
 
+    private long randomDBSuffix = System.currentTimeMillis();
```

Comment:
Okay, well at least correct the variable name and give it a comment on why it's here and is ~safe.